src/main/java/net/geforcemods/securitycraft/items/ModuleItem.java

```diff
@@ -118,7 +118,7 @@ public void addInformation(ItemStack stack, World world, List<ITextComponent> li
 			if(numberOfItemAddons == 0 && numberOfBlockAddons > 0)
 				list.add(new StringTextComponent(TextFormatting.GRAY + ClientUtils.localize("tooltip.securitycraft:module.itemAddons.usage.blocks").replace("#", numberOfBlockAddons + "")));
 
-			if(getNumberOfAddons() > 0) {
+			if(getNumberOfAddons() > 0 && (!getBlockAddons(stack.getTag()).isEmpty()) || !getAddons(stack.getTag()).isEmpty()) {
```

Comment:
No need to check for both `getBlockAddons` _and_ `getAddons`, as the former just automatically converts the `ItemStack`s that `getAddons` would output into `Block`s. Only check `getAddons`.